neqo-transport/src/streams.rs

```diff
@@ -229,7 +264,7 @@ impl Streams {
             }
         }
 
-        self.send.write_frames(priority, builder, tokens, stats);
+	self.send.write_frames(priority, builder, tokens, stats)
```

Comment:
Add the semi-colon back.

```suggestion
	self.send.write_frames(priority, builder, tokens, stats);
```